python/doc/examples/meta_modeling/kriging_metamodel/plot_gpr_isotropic.py

```diff
@@ -70,34 +73,39 @@
 upper = 1000.0
 
 
-def fitKriging(coordinates, observations, covarianceModel, basis):
+def fitGPR(coordinates, observations, covarianceModel, basis):
     """
-    Fit the parameters of a Kriging metamodel.
+    Fit the parameters of a Gaussian Process Regression surrogate model.
     """
     # Define the Kriging algorithm.
```

Comment:
Kriging should be removed from comments too

Reply:
done